src/state/swap/hooks.ts

```diff
@@ -258,11 +266,13 @@ export function useDerivedSwapInfo(platformOverride?: RoutablePlatform): UseDeri
         })
       })
       .catch(error => {
-        console.error(error)
-        unstable_batchedUpdates(() => {
-          setInputError(SWAP_INPUT_ERRORS.UNKNOWN)
-          setLoading(false)
-        })
+        if (error.name !== 'AbortError') {
+          console.error(error)
+          unstable_batchedUpdates(() => {
+            setInputError(SWAP_INPUT_ERRORS.UNKNOWN)
+            setLoading(false)
+          })
+        }
```

Comment:
```suggestion
            setInputError(SWAP_INPUT_ERRORS.UNKNOWN)
            setLoading(false)
        }
```

we dont need this after react18

Reply:
Forgot to remove this. Thanks!